app/routes/meetings/components/MeetingEditor.tsx

```diff
@@ -105,6 +105,8 @@ type MeetingEditorParams = {
 };
 const MeetingEditor = () => {
   const { meetingId } = useParams<MeetingEditorParams>();
+  const [formDirtyFlag, setFormDirtyFlag] = useState(0);
```

Comment:
Do you have any reasoning behind this naming? Didn't find it too intuitive

Also the method of refreshing the form is still kinda hacky - so might be nice with a comment about why this is needed.

```suggestion
  const [reportKey, setReportKey] = useState(0); // Needed to refresh the form component after loading the report content from localStorage
```

Now I'm just throwing out ideas, but I'm assuming this prompt is only supposed to be shown at page load. Is there any neat way that you could prevent loading the report component until the prompt has been adressed? If you could do that and set the correct value before rendering the component, I imagine you wouldn't need the hacky key-changing